scrapers/scrape_ag.py

```diff
@@ -3,17 +3,31 @@
 import scrape_common as sc
 import re
 
-print('AG')
 
 # get latest from list with all bulletins
 d = sc.download('https://www.ag.ch/de/themen_1/coronavirus_2/lagebulletins/lagebulletins_1.jsp')
 
 url = sc.find(r'<a [^>]*href="([^"]+\.pdf)">.+Bulletin.+</a>', d)
 
 # download latest PDF
-d = sc.pdfdownload('https://www.ag.ch' + url, raw=True)
-
+pdf_url = 'https://www.ag.ch' + url
+d = sc.pdfdownload(pdf_url, raw=True)
```

Comment:
use silent=True to avoid output for this download.

Reply:
@metaodi Changed